tools/accuracy_checker/accuracy_checker/adapters/attributes_recognition.py

```diff
@@ -179,6 +179,26 @@ def process(self, raw, identifiers=None, frame_meta=None):
         return result
 
 
+class AgeRecognitionAdapter(AgeGenderAdapter):
```

Comment:
not sure that  inheritance logic is correct (should be reversed as the rule).

Reply:
Updated